app/services/video-encoding-optimizations/video-encoding-optimizations.ts

```diff
@@ -116,19 +116,32 @@ export class VideoEncodingOptimizationService extends PersistentStatefulService<
       profiles = this.state.lastLoadedProfiles;
     } else {
       // try to fetch game-specific profile
-      profiles = await fetch(this.urlService.getStreamlabsApi(`gamepresets/${game.toUpperCase()}`))
-        .then(handleErrors)
-        .then(camelize);
+
+      try {
+        profiles = await fetch(
+          this.urlService.getStreamlabsApi(`gamepresets/${encodeURIComponent(game.toUpperCase())}`),
+        )
+          .then(handleErrors)
+          .then(camelize);
+      } catch (e) {
+        // probably some network error
+        // don't stop here
```

Comment:
Please `console.error` the error.  Silently swallowing exceptions leads to us missing issues down the road.